content/rs/administering/database-operations/migrate-to-active-active.md

```diff
@@ -0,0 +1,96 @@
+---
+Title: Migrate a Database to Active-Active
+description:
+weight: $weight
+alwaysopen: false
+categories: ["RS"]
+---
+With [Active-Active databases]({{< relref "/rs/administering/designing-production/active-active.md" >}}), applications can read and write to the same data set from different geographical locations seamlessly and with latency less than 1 ms, without changing the way the application connects to the database.
+Active-Active databases also provide disaster recovery and accelerated data read-access for geographically distributed users.
+
+If you have data in a single-region RS database that you want to migrate to an Active-Active database,
+you'll need to create a new Active-Active database and migrate the data into the new database as a [Replica Of](https://docs.redislabs.com/latest/rs/administering/active-passive/) the existing database.
+This process will gradually populate the data in the Active-Active database.
+
+Before data migration starts, all data is flushed from the Active-Active database.
```

Comment:
no need in that case, but it also might not be an empty DB

Reply:
@AlonMagrafta are you okay with me taking this text out of the article? If it's an automatic process and I think it might just confuse the user as it confused me.

Reply:
It should be mentioned somewhere on the page as a note, doesn;t have to be here.
Flush is considered a serious operation

Reply:
Ok. We will leave that warning just in case someone decides to use an existing A-A database instead of creating a new one.